versions.json

```diff
@@ -1,4 +1,5 @@
 [
+  "v0.7",
   "v0.6",
   "v0.5",
   "v0.4"
```

Comment:
remove v0.4, we keep lastest 3 version here.

Reply:
done